odl/contrib/torch/operator.py

```diff
@@ -102,7 +102,7 @@ def forward(self, input):
             self.save_for_backward(input)
 
         # TODO: use GPU memory directly if possible
-        input_arr = input.cpu().numpy()
+        input_arr = input.detach().numpy()
```

Comment:
If `input` is on the GPU, this fails. We need both `detach()` and `cpu()`.

Reply:
Do you mean `input.cpu().detach().numpy()`?

Reply:
Yes, or `input.detach().cpu().numpy()`, whatever you like better. Perhaps a little helper `to_numpy` would make that less ugly.